contrib/export/lib.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+import argparse
+
+
+def init_default_argument_parser(prog_desc,
```

Comment:
Why have this in a separate file? Having a single file made it easier to review. Will be true for other community members. And we can take the hash of a single file (to state in the upgrade instructions).

Reply:
I thought it could be nice to have a generic parser so next script's dev will have a thing less to care about